services/api-server/README.md

```diff
@@ -36,6 +36,37 @@ will start the api-server in development-mode together with a postgres db initia
 - http://127.0.0.1:8000/docs: redoc documentation
 - http://127.0.0.1:8000/dev/docs: swagger type of documentation
 
+## Backwards compatibility of the server API
+The public API is required to be backwards compatible in the sense that a [client](https://github.com/ITISFoundation/osparc-simcore-clients) which is compatible with API version `N` should also be compatible with version `N+1`. Because of this, upgrading the server should never force a user to upgrade their client: The client which they have is already compatible with the new server. Note that
+
+```mermaid
+
+flowchart LR
+  subgraph Client
+    direction LR
+    A2(dev branch) .-> B2(master) .-> C2(staging) .-> D2(production)
+    A2(dev branch) ==> B2(master) ==> C2(staging) ==> D2(production)
+  end
+  subgraph Server
+    direction LR
+    A1(dev branch) ~~~ B1(master) ~~~ C1(staging) ~~~ D1(production)
```

Comment:
the flowchart is cool!
perhaps you can annotate the meaning of the arrows (not only in the text) e.g. add notes at the edges with a label as `compatible with` or a legend?

Reply:
I added gitmoji icons to the arrows to indicate the meaning of the different arrows.